rollup.config.js

```diff
@@ -0,0 +1,48 @@
+import babel from 'rollup-plugin-babel';
+import postcss from 'rollup-plugin-postcss';
+import peerDepsExternal from 'rollup-plugin-peer-deps-external';
+import autoprefixer from 'autoprefixer';
+import localResolve from 'rollup-plugin-local-resolve';
+import resolve from 'rollup-plugin-node-resolve';
+import commonjs from 'rollup-plugin-commonjs';
+
+import pkg from './package.json';
+
+const config = {
+  input: 'src/index.js',
+  output: [
+    {
+      file: pkg.browser,
+      format: 'umd',
+      name: pkg.name,
+      globals: {
+        react: 'React',
+        'react-dom': 'ReactDOM',
+        'prop-types': 'PropTypes'
```

Comment:
I think we should drop support for prop-types and forget they ever existed. TypeScript is way better and it doesn't affect production builds as prop-types do (without external plugins to remove them which is not 100% safe as code may depend on them during runtime).